search/intermediate_result.cpp

```diff
@@ -24,7 +24,6 @@ namespace search
 {
 
 /// All constants in meters.
```

Comment:
It's quite arguable what's better - a comment on an individual line or an inline comment.